spec/factories.rb

```diff
@@ -298,6 +298,21 @@ def r_str
     end
   end
 
+  factory(:report) do
```

Comment:
Actually, most of the parenthesis here are unnecessary. Please see other factory definitions...